openwisp_radius/templates/custom_password_reset_email.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n %}{% autoescape off %}
+{% blocktrans %}You're receiving this email because you requested a password reset for your user account at
+{{ site_name }}.{% endblocktrans %}
+
+{% trans "Please go to the following page and choose a new password:" %}
+{% block reset_link %}
+{# Password reset confirm url #}
+http://example.com/password/reset/{{uid}}/{{token}}
```

Comment:
I didn't notice before, but this URL is hardcoded, which is not good because it cannot be edited by users and it won't work.

I guess this will be the URL of the captive page where the user will complete the password reset right?

I think this URL should be configurable with a django setting but should be different for each organization, because the captive page changes for each organization.

I would use a setting like the following:

```python
PASSWORD_RESET_URLS = {
    'default': 'https://example.com/{slug}/password/reset/{uuid}/{token}'  # fallback in case the specific org page is not defined
    # use the uuid because the slug can change
    'a8697849-6f9c-443a-b530-9dde3a910214': ''https://organization1.com/{slug}/password/reset/{uuid}/{token}',
    '463bdcc3-7bb8-479e-86dc-eb3f0c7eb50d: ''https://organization2.com/{slug}/password/reset/{uuid}/{token}'
}
```

You then retrieve the `Organization` in the view from the slug and use it to get the URL:

```python
organization = Organization.objects.get(slug=slug)
password_reset_url = settings.PASSWORD_RESET_URLS.get(str(organization.id))
if not password_reset_url:
    # always provide a default value
    password_reset_url = PASSWORD_RESET_URLS.get('default', DEFAULT_PASSWORD_RESET_URL)
```

Keep in mind this is almost pseudo-code.

Reply:
Can't we modify the organisations model and add a password_reset_url field or some front-end url field to it. I think it will be a better way of doing it.

Reply:
no, because the module that defines the `Organization` model, openwisp-users, does not know anything about this feature that is specific to the radius module and it should not know.
In the near future we can add something easier to configure like we have for the organization shared secret for openwisp controller or the organization radius API token for the radius module, we could have a model that is related to `Organization`, which stores all the configuration of a captive page for a specific org, but it's too early to do that now.

However, once we complete this, we should create the issue to do this improvement, so remind me please if I forget.